stanford-earth/stanford_earth_helper
# EARTH-689: Subsite Functionality

# READY FOR REVIEW

# Summary
- The first version of subsite functionality.

# Needed By (Date)
- Thursday February 15th, 2018

# Urgency
- Moderate

# Steps to Test

1. Check out branch EARTH-689 of SE3_BLT
2. Run composer update or install
3. Run `blt sync:refresh` or something equivalent 
4. Log in as an administrator
5. Create a couple of parent subsites and a few children
6. Validate that the functionality meets the criteria of EARTH-656, EARTH-658, and EARTH-601

# Associated Issues and/or People
- EARTH-656
- EARTH-658
- EARTH-601
- EARTH-689

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
